resolwe/flow/tests/test_expression_filters.py

```diff
@@ -153,6 +174,7 @@ def test_templatetags(self):
         self.assertEqual(data.output['safe'], 'this')
         self.assertEqual(data.output['description_text'], 'This is test Data object.')
         self.assertEqual(data.output['list_description_text'], 'This is test Data object.')
+        self.assertEqual(data.output['input_id'], 1)
```

Comment:
Change `1` to `inputs_input_data.pk` as this can change over time.

Reply:
Changed